deadline_cloud_extension/DeadlineCloud.pyp

```diff
@@ -4,12 +4,7 @@ import sys
 root = os.path.dirname(__file__)
 sys.path.insert(0, os.path.join(root, 'modules'))
 
-if 'cinema4d_submitter' not in sys.modules.keys():
-    python_path = os.getenv('DEADLINE_CLOUD_PYTHONPATH')
```

Comment:
Considering this has an impact on the environment variable that we set, can we update the instructions as well? I mean [over here](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/blob/f26c9ac7f213f6be9719226bfcc8965caf328d6c/DEVELOPMENT.md?plain=1#L41)

Reply:
Thanks for updating!